scripts/saml_script/README.md

```diff
@@ -0,0 +1,29 @@
+# Dell sample script 
+Sample script in Python to get SAML token from Okta. SAML token can be written to named pipe. 
+In this example, /tmp/samlPipe needs to be created first.
+Provider will read SAML token from that pipe and use it to get Dell Identity token.
+
+
+# Getting Started
+Install prerequisites to run Python script in saml_script directory. 
+Copy either win or lin file to saml_get_token_okta_idp.py as per the platform. 
+
+Update userid/password in saml_get_token_okta_idp.py.
```

Comment:
Can you give the line numbers in the script where they need to update these values?